### PR TITLE
Adding the 60 days period

### PR DESCRIPTION
**Pull request by plehegar**
per Process 2018

### CODE REVIEW

**Comment by plehegar** [comment]
Part of https://github.com/w3c/modern-tooling/issues/71

File: process/charter.html [hunk 1]
```diff
@@ -362,27 +362,12 @@ <h4>Updating the Charter End Date in the Charter</h4>
...
-Committee review of every new or substantively modified Working Group
-or Interest Group charter. The Director is NOT REQUIRED to solicit
-Advisory Committee review prior to a charter extension or for minor
-changes."</p>
+<p>Per <a href="https://www.w3.org/Consortium/Process/#CharterReview">section 5.2.3
+of the W3C Process</a>, The Director must solicit Advisory Committee review of every new or substantively modified Working Group or Interest Group charter.
+The Director is not required to solicit Advisory Committee review prior to a charter extension or for minor changes. The review period must be at least 28 days.
+Any Advisory Committee representative may request an extended review period, if submitted with a Member’s comments in response to the Call for Review (see <a href="https://www.w3.org/new-doc-from-template?location=%2FTeam%2Ftemplates&amp;template=%2Fafs%2Fw3.org%2Fpub%2FWWW%2FTeam%2FTemplates%2Fcharter-extension.html&amp;submit=Continue...">extension template</a>). Upon receipt of
+any such request, the Director must ensure that the Call for Participation for the Working Group occurs at least 60 days after the Call for Review of the charter.
```
Comment (koalie) [inline process/charter.html]: s/the Working Group/the Working Group or Interest Group/

File: process/charter.html [hunk 2]
```diff
@@ -638,6 +623,12 @@ <h3>6.3 Organizing the Call for Participation</h3>
 make sure the charter is public (since 2007, charters are public during AC review) and any final edits (e.g., addition of
 link to page for joining the group) have been made.</p>
 
+<p>Per <a href="https://www.w3.org/Consortium/Process/#CharterReview">section 5.2.3
+  of the W3C Process</a>, any Advisory Committee representative may request an extended review period, if submitted with a Member’s comments in response to the Call for Review.
+  Upon receipt of
+  any such request, the Director must ensure that the Call for Participation for the Working Group occurs at least 60 days after the Call for Review of the charter.
```
Comment (koalie) [inline process/charter.html]: s/the Working Group/the Working Group or Interest Group/

**Comment by plehegar** [comment]
@koalie nice catch! Actually, this is only applicable to Working Groups if you read carefully the new Process. So, I made this clearer in the document.

**Pull request merged**